File: app/src/main/java/com/ndipatri/solarmonitor/providers/solarUpdate/dto/solaredge/CurrentPower.java

package com.ndipatri.solarmonitor.providers.solarUpdate.dto.solaredge;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CurrentPower {

    @SerializedName("power")
    @Expose
    private Double power;

    public Double getPower() {
        return power;
    }

    public void setPower(Double power) {
        this.power = power;
    }

}

File: app/src/main/java/com/ndipatri/solarmonitor/providers/solarUpdate/dto/solaredge/CurrentPower.kt
package com.ndipatri.solarmonitor.providers.solarUpdate.dto.solaredge

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CurrentPower {

    @SerializedName("power")
    @Expose
    var power: Double? = null

}
